react/components/ShippingSimulator/index.js

```diff
@@ -19,7 +19,7 @@ class ShippingSimulator extends Component {
     intl: intlShape.isRequired,
     client: PropTypes.object,
     skuId: PropTypes.string,
-    seller: PropTypes.number,
+    seller: PropTypes.string,
```

Comment:
In the [docs](https://github.com/vtex-apps/store-components/blob/master/react/components/ShippingSimulator/README.md#configuration) it says this prop is required. You could change it here (or there)

(Might be a good time to include the other ones there)

Reply:
Done!